app/views/capture.njk

```diff
@@ -45,13 +72,13 @@
                 <th scope="col" class="govuk-table__header govuk-body-s">Debt type</th>
                 <th scope="col" class="govuk-table__header govuk-table__header--numeric govuk-body-s">Date of Discovery</th>
                 <th scope="col" class="govuk-table__header govuk-body-s">Created by</th>
-                <th scope="col" class="govuk-table__header">Status</th>
+                <th scope="col" class="govuk-table__header govuk-body-s">Status</th>
                 <th scope="col" class="govuk-table__header"></th>
               </tr>
             </thead>
             <tbody class="govuk-table__body">
               {% for data in captureData %}
-                <tr class="govuk-table__row">
+                <tr class="govuk-table__row capture-datasets">
                   <td scope="col" class="govuk-table__cell govuk-body-s">{{ data.schemes.name }}</td>
                   <td scope="col" class="govuk-table__cell govuk-body-s">{{ data.frn }}</td>
                   <td scope="col" class="govuk-table__cell govuk-body-s">{% if data.reference %}{{ data.reference }}{%else%}Manual enrichment{%endif%}</td>
```

Comment:
On my local testing using the `payload` given in the `README`, I am seeing:

-  for the col `Agreement number` I am seeing `Manual enrichment` despite the `payload` having an `Agreement number`
- for `Net value` a value of `£0.00`

How to recreate:

- send 1 `README payload` to `ffc-pay-debt-data`
- enrich it via `/enrich`
- view `/capture`

Will attach a screenshot to the ticker in Jira

Reply:
Also, use whitespace to help break up this `if-else` similar to lines 90 - 94

Reply:
Resolved this with Steve by updating the `get-debts` query to filter out the applications which have a `reference` of `Manual enrichment`